connector/saml/saml.go

```diff
@@ -81,8 +81,10 @@ type Config struct {
 	//
 	// https://www.oasis-open.org/committees/download.php/35391/sstc-saml-metadata-errata-2.0-wd-04-diff.pdf
 
-	Issuer string `json:"issuer"`
-	SSOURL string `json:"ssoURL"`
+	// Issuer refers to the SP issuer
+	Issuer    string `json:"issuer"`
+	SSOIssuer string `json:"ssoIssuer"`
```

Comment:
Refer to the issue I filed. In a nutshell, there appears to be some confusion between the SP issuer and IDP issuer.